### ISSUE TITLE
SSURGO warning in CO Storm Runoff 

### ISSUE THREAD

**Issue by amedenblik** [USGS-WiM/StreamStats#1396]
@USGSPMM3 was using the CO storm runoff models at this point (39.34897, -104.97683) and encountered this error:

![image](https://user-images.githubusercontent.com/40237491/182878344-08d3e834-19f8-4d8e-a32c-b98959f6622c.png)

I see that this error comes from here: https://github.com/USGS-WiM/StreamStats/blob/master/src/Controllers/StormRunoffController.ts#L255

Pete was still able to continue the workflow and get model results, despite seeing the error. 

Questions:
1. Is this the intended functionality? 
2. What is the extent of the map service (https://gis.usgs.gov/sciencebase2/rest/services/Catalog/5b96f40ce4b0702d0e8272bf/MapServer/0)? How much of CO does it cover? I tried viewing the map service [here](https://www.arcgis.com/home/webmap/viewer.html?url=https%3A%2F%2Fgis.usgs.gov%2Fsciencebase2%2Frest%2Fservices%2FCatalog%2F5b96f40ce4b0702d0e8272bf%2FMapServer&source=sd) but didn't see any of the polygons.

**Comment by amedenblik** [comment]
Follow up on question #2: I was able to view the map layer in ArcMap: 

![image](https://user-images.githubusercontent.com/40237491/182882100-80fb8aad-9acb-419a-849f-287790f7e90f.png)

**Comment by tgross-usgs** [comment]
@USGSPMM3 This is the intended functionality. We discovered after generating the dataset that SSURGO doesn't have data available in mountain peak areas and therefore the results for curve number are skewed in those areas. In hindsight we should have used STATSGO data but it was already published when we made the discovery and haven't bothered to reprocess the data. However, those using RCN in Colorado are most likely using it with our TR55 tool which only applies to small basins anyway and I hope that there aren't too many affected by the data gap.
